openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -131,39 +134,48 @@ All OAuth 2.0 Grant Types and extensions mechanisms can be used in conjunction w
 
-* multiple Credentials of different types/doctypes bound to the same proof, or 
-* multiple Credentials of the same type/doctype bound to different proofs, or
-* multiple Credentials of different types/doctypes bound to different proofs.
+This specification allows for the issuance of Verifiable Credentials through two types of endpoints: the Single Credential Endpoint and the Batch Credential Endpoint.
 
-In the course of the authorization process, the Credential Issuer MAY also request Credential presentation as a means to authenticate or identify the End-User during the issuance flow, as illustrated in (#use-case-5).
+1. Single Credential Endpoint: This endpoint is used when a single Verifiable Credential is to be issued. The request to this endpoint includes the necessary parameters for the issuance of one credential. The response from this endpoint will contain the issued Verifiable Credential.
+
+2. Batch Credential Endpoint: This endpoint is used when multiple Verifiable Credentials need to be issued at once. The request to this endpoint includes the necessary parameters for the issuance of multiple credentials. The response from this endpoint will contain all the issued Verifiable Credentials.
 
-At its core, this specification is Credential format agnostic and allows implementers to leverage specific capabilities of Credential formats of their choice. Multiple Credential formats can be used within the same transaction. 
+When dealing with multiple Credentials, the following combinations are possible:
 
-The specification achieves this by defining the following:
+1. Multiple Credentials with different Credential Format bound to the same Cryptographic Holder Binding.
+2. Multiple Credentials with the same Credential Format bound to different Cryptographic Holder Binding.
+3. Multiple Credentials with different Credential Format bound to different Cryptographic Holder Binding.
 
-* Extension points to add Credential format specific parameters or claims in the Credential Issuer metadata, Credential Offer, Authorization Request, Credential Request and Batch Credential Request,
-* Credential format identifiers to identify Credential format specific set of parameters and claims to be applied at each extension point. This set of Credential format specific set of parameters and claims is referred to as a "Credential Format Profile" in this specification.
+These combinations allow for flexibility in how Verifiable Credentials are issued, accommodating a variety of use cases and requirements. The choice between using the Single Credential Endpoint or the Batch Credential Endpoint depends on the specific use case. If only one Verifiable Credential needs to be issued, the Single Credential Endpoint would be used. If multiple Verifiable Credentials need to be issued at once, the Batch Credential Endpoint would be more efficient.
```

Comment:
The open question is: Will this PR be merged before or after the single credential endpoint is removed? 
https://github.com/openid/OpenID4VCI/issues/18